dora/core/common/src/main/java/alluxio/wire/FileInfo.java

```diff
@@ -336,6 +338,22 @@ public Map<String, byte[]> getXAttr() {
     return mXAttr;
   }
 
+  /**
+   * @return the ufs name
+   */
+  @Nullable
+  public String getUfsName() {
+    return mUfsName;
+  }
+
+  /**
+   * @return the content hash
+   */
+  @Nullable
+  public String getContentHash() {
```

Comment:
Does it has to be string? `byte[]` should be more efficient as we can save the cost to de/serialize the output of hashing algorithnms.